<launch>
    <!-- Motor node -->
    <include file="$(find xycar_motor)/launch/xycar_motor.launch" />
    
    <!-- USB Camera node -->
    <include file="$(find usb_cam)/launch/xycar_cam.launch" />
    
    
    <node name="h_drive" pkg="my_hough" type="pid_hough_drive.py" output="screen"/>
</launch>